### DISCUSSION TITLE
Feedback: Higher-order resolver

### DISCUSSION THREAD

**Discussion by patricklafrance** [mswjs/mswjs.io#361] [Q&A] [answered]
- [Open page](https://mswjs.io/docs/recipes/higher-order-resolver)

## Feedback

That's a great example but it would be even better if the example was available in TypeScript!

**Comment by kettanaito** [comment, marked as answer]
Hi, @patricklafrance. Thank you for the feedback!

You can find that example in TypeScript on the [Usage with TypeScript](https://mswjs.io/docs/best-practices/typescript#higher-order-response-resolvers) best practice. 

I hope to improve the code snippets across the docs to feature TypeScript in time but I don't have that time right now.
